hummingbird/ml/operator_converters/_linear_implementations.py

```diff
@@ -17,7 +17,7 @@ class LinearModel(BaseOperator, torch.nn.Module):
     def __init__(self, coefficients, intercepts, device, classes=[0], multi_class=None, is_linear_regression=False):
         super(LinearModel, self).__init__()
         self.coefficients = torch.nn.Parameter(torch.from_numpy(coefficients), requires_grad=False)
-        self.intercepts = torch.nn.Parameter(torch.from_numpy(intercepts), requires_grad=False)
+        self.intercepts = torch.nn.Parameter(torch.from_numpy(intercepts).view(-1), requires_grad=False)
```

Comment:
why suddenly the `view(-1)`'s ?  Is this related to TVM? or the container changes?
or should this be yet a 3rd PR?  :-D

Reply:
No no this is because of TVM. TVM is really sensible to tensor shapes.

Reply:
Interesting!  So..were these "wrong" before and we were just getting lucky?

Reply:
PyTorch is a bit more flexible regarding shapes and types. TVM is more compiler-style and requires shapes and types to perfectly match. I think that some operators where we still fail for TVM have the same kind of problem, so I hope they can be fixed quickly.